lib/player.js

```diff
@@ -4100,6 +4053,18 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     return ret;
   }
 
+  /**
+   * Changes low latency configuration settings on the Player.
+   *
+   * @param {!Object} config This object should follow the
+   *    {@link shaka.extern.PlayerConfiguration} object.  Not all fields
+   *    need to be set; unset fields retain their old values.
+   * @export
+   */
+  configurationForLowLatency(config) {
+    this.lowLatencyConfig_ = config;
+  }
```

Comment:
Shouldn't we apply this config immediately in case it's called during LL playback?